src/error.rs

```diff
@@ -187,6 +187,7 @@ impl ApiError {
 
     pub fn render_404<B>(res: ServiceResponse<B>) -> Result<ErrorHandlerResponse<B>> {
         // Replace the outbound error message with our own.
+        dbg!("Returning 404...");
```

Comment:
Probably can remove a lot of these dbg!, since we've isolated why the tests weren't connecting up. They'll probably just be noise later.